src/components/EventCard/index.jsx

```diff
@@ -0,0 +1,20 @@
+import PrimaryButton from "../PrimaryButton";
+
+export default function EventCard({ event }) {
+  return (
+    <div className="flex-row w-full">
+      <div className="flex w-full">
+        <h4 className="flex-initial">{event.Time}</h4>
+        <h2 className="flex-auto ml-4">{event.Header}</h2>
+        <h4 className="flex-initial justify-self-end">{event.Date}</h4>
+      </div>
+      <div className="flex">
+        <p>{event.Description}</p>
```

Comment:
There should be some sort of spacer in between these, so they don't rub against each other